release: 0.4.5

Kurzdor opened the pull request:
Changes:

- Properly handle `autoSize` for `LabelEx`
- Disable grab cursor when mouse is over component and drag is disabled
- Add `TEXTFIELD_AUTOSIZE` constants exports to python-side API
- Add `.gitignore` with WG .swc files 
- Built swf and wotmod included

Commit:
fix(guiflash): handle textfield autosize properly

Commit:
chore(guiflash): add gitignore

Commit:
chore(guiflash): add textfield autosize constants to python exports

Commit:
fix(guiflash): don't show grab cursor when drag is disabled

Commit:
fix(guiflash): this will break border and tooltip if drag is disabled and not neccessary here

Commit:
release(guiflash): 0.4.5

CH4MPi reviewed (changes requested):
- missing class (object) for TEXTFIELD_AUTOSIZE class. see already existing classes
It will be changed with 0.4.6

CH4MPi reviewed (approved):
needs a recompile. I will make this later today after some testing

fusi0n committed:
v0.4.6 for WoT client v1.17 and higher
- properly handle autoSize for LabelEx (thanks to Kurzdor)
- disable grab cursor when mouse is over component and drag is disabled (thanks to Kurzdor)
- add TEXTFIELD_AUTOSIZE constants exports to python-side API (thanks to Kurzdor)
- version number increased to 0.4.6
- gambiter.guiflash_0.4.6.wotmod included

fusi0n committed:
v0.4.6 for WoT client v1.17 and higher
- properly handle autoSize for LabelEx (thanks to Kurzdor)
- disable grab cursor when mouse is over component and drag is disabled (thanks to Kurzdor)
- add TEXTFIELD_AUTOSIZE constants exports to python-side API (thanks to Kurzdor)
- version number increased to 0.4.6
- gambiter.guiflash_0.4.6.wotmod included
